triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Server.scala

```diff
@@ -414,7 +414,7 @@ object Server {
         }
         .receiveSignal {
           case (_, PostStop) =>
-            binding.unbind()
+            discard[Future[akka.Done]](binding.unbind())
```

Comment:
One of the neat things about `discard`'s signature is that you can specify exactly _what_ is being discarded, which may prompt a discussion about whether it _should be_.

Reply:
Good catch! That definitely looks wrong. If you want to leave fixing this to a separate PR, maybe add a TODO.

Reply:
It would also make sense to update `com.daml.scalautil.Statement#discard` so it logs an error or warning if passed `Future` fails, else we just drop the error without any notificaiton.

Reply:
In any case when unbinding, you should return the `Future` to the client, so it can subscribe for the `Future` completion.

Reply:
cdc2e247ab17e5c3133b5a21c000025b511298c0

Reply:
Follow-up issue #7086